index.html

```diff
@@ -0,0 +1,222 @@
+        feature or website Look through some of my work and experience! If you like what you see 
+        and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <p><br><a class="intro-connect btn-action-event">LET’S CONNECT</a><br></p>
+    <ul class="view-social-midea">
+        <li class="social-icon">
+            <a class="btn-action-event"><img src="./assets/images/social-icon/twitter-icon.png" alt="twitter"></a>
+        </li>
+        <li class="social-icon">
+            <a class="btn-action-event"><img src="./assets/images/social-icon/linkedin icon.png" alt="linkedin"></a>
+        </li>
+        <li class="social-icon ">
+            <a><img src="./assets/images/social-icon/m-icon.png" alt="m"></a>
+        </li>
+        <li class="social-icon ">
+            <a><img src="./assets/images/social-icon/github-icon.png" alt="github"></a>
+        </li>
+        <li class="social-icon ">
+            <a><img src="./assets/images/social-icon/clip-icon.png" alt="fingers"></a>
+        </li>
+    </ul>
+    <span class="p-more-information ">
+        <a class="moresee-link btn-action-event">Get my resume</a>
+        </span>
+        <ul class="expand-ul"><li><a class="expand-ula">Languages</a></li><li><input type="image" src="./assets/images/arrowdown.png" class="bn-img-down"></li></ul>
+        <ul class="language-btn">
+           <li><img src="./assets/images/js.png" title="js" alt="javascript icon"> Javascript</li>
+        </ul>
+        <ul class="language-btn">
+            <li><img src="./assets/images/css.png" title="js" alt="javascript icon"> CSS
+            </li>
+        </ul>
+        <ul class="language-btn">
+            <li>
+            <img src="./assets/images/html.png" title="js" alt="javascript icon"> Html
+        </li>
+        </ul>
+        <ul class="job-experience">
+            <li class="job-experience-content">Frameworks</li><li class="job-experience-content">&#62;</li>
+        </ul>
+        <ul class="job-experience">
+            <li class="job-experience-content">Skill</li><li class="job-experience-content">&#62;</li>
+        </ul>
+      </span>
+      <span class="aboutmyself-content about-getintouch">
+        <h1 class="aboutmyself-text">Contact me</h1>
+        <p class="aboutmyself-desc">
+            If you have an application you are interested in developing, a feature that you 
+            need built or a project that needs coding.
+             I’d love to help with it 
+        </p>
+        <input class="input-bg aboutmyself-input" type="text" placeholder="Osman">
+        <input type="email" class="input-bg aboutmyself-input" placeholder="osman@gmail">
+        <textarea class="input-bg aboutmyself-message" cols="100" placeholder="Describe yourself here..."></textarea>
+        <input type="submit" class="input-bg aboutmyself-getintouch" value="Get in touch">
+      </span>
+    </div>
+</section>
+        
```

Comment:
### The comments below are CSS-related changes.

- [ ] Please be aware that the mobile screen size is a range of **min 375px - 786px** currently your website looks amazingly responsive on screen resolution less than **420px** 👌, Pease focus on the responsiveness on the resolution **min 375px - 786px** instead.

![image](https://user-images.githubusercontent.com/68030297/219853635-680fbecf-e50b-4cac-9cee-a97a7547795d.png)